client/source-map-support.js

```diff
@@ -33,12 +30,14 @@ async function rewriteTraceLine (trace: string): Promise<string> {
 
   const mapPath = `${filePath}.map`
 
+  const fetch = require('unfetch')
```

Comment:
Is there any effect loading modules like this? I hope webpack still see them just like before.

Reply:
The weird thing is, it does work for source-map, but not for unfetch 🤷‍♂️